health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/hooks/services/createUpdatePlanProject.js

```
@@ -207,27 +225,24 @@ const createUpdatePlanProject = async (req) => {
+          CampaignDetails: {
             tenantId: tenantId,
-            ids: [
-                campaignId
-            ]
-          }
-        })
-        
-        const prevSelectedBoundaries = campaignObjectForBoundary?.boundaries
+            ids: [campaignId],
+          },
+        });
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)